src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -225,10 +227,9 @@ export const AlterationTypeConstants = {
     STRUCTURAL_VARIANT: 'STRUCTURAL_VARIANT',
 };
 
-// only show TREATMENT_RESPONSE in the plots tab for now
-// TODO: apply to all generic assay profiles when front-end implementation finish
-export const GenericAssayTypeConstants = {
+export const GenericAssayTypeConstants: { [s: string]: string } = {
```

Comment:
lets make a little typescript change here.

step one: remove the typing (`: { [s: string]:string }`)
step two:

Reply:
same as below.